store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/retriever/AccumuloSetRetriever.java

```diff
@@ -175,19 +184,19 @@ private void addToBloomFilter(final Object vertex, final BloomFilter filter) thr
         private Iterator<Element> iterator;
         private Element nextElm;
 
+        @SuppressWarnings({"unchecked", "rawtypes"})
```

Comment:
What's the rationale for adding this? Can't this warning's be fixed? There are thousands of rawtype warnings in the codebase which are not suppressed, it might be better to keep that consistent.

Reply:
I do think that another ticket should be made to cover fixing as many warnings as possible in Gaffer 2